src/ui/public/url/redirect_when_missing.js

```diff
@@ -57,7 +58,9 @@ export function RedirectWhenMissingProvider($location, kbnUrl, Promise) {
       url += (url.indexOf('?') >= 0 ? '&' : '?') + `notFound=${error.savedObjectType}`;
 
       toastNotifications.addWarning({
-        title: 'Saved object is missing',
+        title: i18n.translate('common.ui.url.savedObjectsIsMissingNotificationMessage', {
```

Comment:
Please inject `i18n` service here
https://github.com/elastic/kibana/blob/7b27cbe0282ebf32d932fc6bdc561d8148586f68/src/ui/public/url/redirect_when_missing.js#L31
and use it.